nupic/research/frameworks/dynamic_sparse/models/main.py

```diff
@@ -310,10 +278,12 @@ def setup(self):
                     module.apply_mask()
                     module.save_num_params()
 
-        # TODO: remove logging to separate class
+        self.logger = SparseLogger(self)
+        # TODO: is this still required?
```

Comment:
@mvacaporale Not sure what this was supposed to do, I didn't get the logic here. Can you please check if it is still required?

Reply:
Which line are you talking about? The one with `log_magnitude_vs_coactivations`?

Reply:
The line 282 to 284 there